sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/CosmosConfigSpec.scala

```diff
@@ -64,7 +64,7 @@ class CosmosConfigSpec extends UnitSpec with BasicLoggingTrait {
 
           endpointConfig.endpoint shouldEqual sampleProdEndpoint
 
-          val aadAuthConfig = endpointConfig.authConfig.asInstanceOf[CosmosAadAuthConfig]
+          val aadAuthConfig = endpointConfig.authConfig.asInstanceOf[CosmosServicePrincipalAuthConfig]
```

Comment:
Should this variable be renamed servicePrincipleAuthConfig as well ?

Reply:
Done